bigquery-service/src/main/java/com/github/saintukrainian/bigqueryservice/entities/Hospital.java

```diff
@@ -0,0 +1,15 @@
+package com.github.saintukrainian.bigqueryservice.entities;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
```

Comment:
Do you need only getters and setter generated by lombook, how about equals, hash code, probably these methods will be better to have overriden too, what do you think?